elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/ElideAutoConfiguration.java

```diff
@@ -135,15 +136,4 @@ public Swagger buildSwagger(EntityDictionary dictionary, ElideConfigProperties s
 
         return swagger;
     }
-
-    /**
-     * Configure the QueryEngineFactory that the Aggregation Data Store uses.
-     * @param entityManagerFactory Needed by the SQLQueryEngine
-     * @return a SQLQueryEngineFactory
-     */
-    @Bean
-    @ConditionalOnMissingBean
-    public QueryEngineFactory buildQueryEngineFactory(EntityManagerFactory entityManagerFactory) {
```

Comment:
We should provide a bean that allows users to override the QueryEngine bean that is used.